src/components/date-picker/DatePicker.js

```diff
@@ -16,6 +16,7 @@ import PlainButton from '../plain-button';
 import Tooltip from '../tooltip';
 
 import { convertDateToUnixMidnightTime } from '../../utils/datetime';
+import { get as getCookie } from '../../api/box-edit/cookies';
```

Comment:
I don't think we want to use a Box Edit API helper in a common component. We probably should find another way to get the value rather than via a cookie.

Reply:
Would a better option be to get the `intl.locale`?

Reply:
`intl.locale` only contain language info like `en` or `zh`. But here we need country info like `en-US` or `en-UK`. US first day is Sunday and CA is Monday.

Reply:
https://git.dev.box.net/Box/EndUserApp/pull/4669

Reply:
Whatever we do here, this needs to work with public APIs (partner platforms) and safely fall back when the locale doesn't have the country code.

Reply:
Also see https://github.com/tc39/ecma402/issues/6

Reply:
cc @ehoogerbeets @jeeyi